packages/@lwc/integration-karma/test/light-dom/scoped-slot/x/row/row.html

```diff
@@ -0,0 +1,9 @@
+<template lwc:render-mode="light">
+    <p>
+        <template for:each={row} for:item="field">
+            <span key={field.id}>
```

Comment:
Side notes, there isn't any test checking if `key` can be assigned to the `<slot>` with the `lwc:slot-bind` directive.